src/nyx/main_nyxus.cpp

```diff
@@ -93,6 +101,8 @@ int main (int argc, char** argv)
 		case 3:		// Memory error
 			std::cout << std::endl << "Memory error" << std::endl;
 			break;
+		case 4:
+			std::cout << std::endl << "Arrow not enabled" << std::endl;
```

Comment:
Maybe it's worth adding a couple of words how to enable, reinstall, or reset something? If Arrow libraries aren't in place, the executable/library wouldn't even run, so a user may be clueless why Arrow may still not be enabled.